tools/CMakeLists.txt

```diff
@@ -5,3 +5,4 @@
 add_subdirectory(model)
 add_subdirectory(pipeline)
 add_subdirectory(link-for-translation)
+add_subdirectory(invalidate)
```

Comment:
`pipeline/invalidate`?

Reply:
done that, but do you want to rename them with a particular convetion? eg revng pipeline invalidate? that requires manipulating the driver as well.